vanilla-installer/theme.py

```diff
@@ -40,37 +40,48 @@ def is_dark(to_dark: bool = None) -> bool:
         return open(FILE, "w", encoding="utf-8").write("dark")
     return open(FILE, encoding="utf-8").read() == "dark"
 
+# colors from catppuccin latte and mocha https://github.com/catppuccin/catppuccin
```

Comment:
Probably worth adding a mention in credits.md for this one. Speaking of credits, is it worth adding a button for that?

Reply:
The icons are also from feathericons. we can add that to credit.md too